src/components/CoinChart/CoinChartArea.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from "react";
+
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+import { SDiv } from "@styles";
+
+import CoinChartCanvas from "./CoinChartCanvas";
+import SelectDurationChip from "./CoinSelectDurationChip";
+
+const CoinChartArea = ({
+  coinCurrency,
+  isGreen,
+  chartData,
+  handleClickChangeChipQuery,
+}) => {
+  // chip 버튼을 클릭하면, query에 맞게 UI로 보여줄 텍스트 데이터를 매칭했습니다.
+  const queries = [
+    ["max", "전체"],
+    ["365", "1년"],
+    ["31", "1개월"],
+    ["7", "일주일"],
+    ["1", "1일"],
+  ];
```

Comment:
```suggestion
  const queries = {
    "전체" : "max",
    "1년" : "365",
    "1개월": "31",
    "일주일": "7",
    "1일": "1",
  };
```

이 낫지 않을까요? 아래에 나오는 queries[idx][1] 이런 표현들이 명시적이지 않아 보여서요! 배열 첫 번째가 어떤 역할인지 바로 보이지가 않을 것 같네요